runtime/interpreter/value.go

```diff
@@ -14672,6 +14672,9 @@ func (v *CompositeValue) Transfer(
 			panic(ExternalError{err})
 		}
 
+		elementMemoryUse := common.NewAtreeMapPreAllocatedElementsMemoryUsage(v.dictionary.Count())
```

Comment:
`CompositeValue`'s `elementSize` is always 0, meaning variable-sized.  

`NewAtreeMapPreAllocatedElementsMemoryUsage()` API is changed to receive `elementSize`.  Details in the previous comment.   

```suggestion
		elementMemoryUse := common.NewAtreeMapPreAllocatedElementsMemoryUsage(v.dictionary.Count(), 0)
```